### PR TITLE
Avoid rendering invalid JSON characters in response payloads

### PR DESCRIPTION
**Pull request by fgalan**
**DO NOT DELETE BRANCH AFTER MERGE** (note that this PR uses release/0.26.1 as base, we need the branch to merge also in develop at the end).

As a side fix, also avoid escaping / in callback field in GET /v2/subscriptions and GET /v2/subscriptions/<subId> operations.

All unit and harness test passing.

@kzangeli @crbrox

### CODE REVIEW

File: src/lib/common/JsonHelper.cpp [hunk 1]
```diff
@@ -48,12 +48,20 @@ std::string toJsonString(const std::string& input)
      * but as long as we support NGSIv1, it is better to have the check (e.g. a newline could be 
      * used in an attribute value using XML). Even removing NGSIv1, we have to ensure that the 
      * input parser (rapidjson) doesn't inject not supported JSON characters in the DB (this needs to be
-     * investigated in the rapidjson documentation) */
+     * investigated in the rapidjson documentation)
+     *
+     * JSON specification is a bit obscure about the need of escaping / (what they call 'solidus'). The
+     * picture at JSON specification (http://www.json.org/) seems suggesting so, but after a careful reading of
+     * https://tools.ietf.org/html/rfc4627#section-2.5, we can conclude it is not mandatory. Online checkers
+     * such http://jsonlint.com confirm this. Looking in some online discussions
```
Comment (kzangeli) [inline src/lib/common/JsonHelper.cpp]: such as

Comment (fgalan) [inline src/lib/common/JsonHelper.cpp]: Fixed in 01f659c

**Comment by crbrox** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**